src/lang/db/semantic.rs

```diff
@@ -45,12 +46,41 @@ pub trait LsSemanticGroup: Upcast<dyn SemanticGroup> + SemanticGroup {
             .find_map(|node| lookup_item_from_ast(self.upcast(), module_file_id, node))
     }
 
+    /// Like [`LsSemanticGroup::collect_lookup_items_stack_in_file`], but also checks parent files nodes using code mappings.
+    fn collect_lookup_items_stack(&self, node: &SyntaxNode) -> Option<Vec<LookupItemId>> {
```

Comment:
The name is not describing what it actually does. Also the description in comment is not accurate